malwarefront/config-overrides.js

```diff
@@ -1,7 +1,76 @@
-const { override, babelInclude, removeModuleScopePlugin } = require('customize-cra');
+const { override, babelInclude, removeModuleScopePlugin, addWebpackPlugin, addWebpackModuleRule, addWebpackResolve } = require('customize-cra');
+const fs = require("fs");
 const path = require('path');
+const webpack = require('webpack');
 
-module.exports = override(
-  removeModuleScopePlugin(),
-  babelInclude([path.resolve('src'), path.resolve(__dirname, '../plugins')])
-);
\ No newline at end of file
+const pluginPackagePrefix = "@malwarefront-plugin";
+const pluginsIndexFile = path.resolve(require.resolve("@malwarefront/extensions"), '..', 'plugins.js');
+
+function findInstalledPlugins() {
+    let modules = {};
+    let modulesDir = path.resolve(__dirname, "node_modules", pluginPackagePrefix);
+    if(!fs.existsSync(modulesDir))
+        return {}
+    for(let pluginName of fs.readdirSync(modulesDir)) {
+        let realPath = path.resolve(modulesDir, pluginName);
+        try {
+            realPath = path.resolve(modulesDir, fs.readlinkSync(realPath));
```

Comment:
Is there any disadvantage to using a symlink? Why do we have to read it?

Reply:
Some parts of Webpack configuration (e.g. babel configuration) don't recognize symlinks, they're working only with already resolved paths.